build_tools/aws-sdk-code-generator/spec/protocol-tests-ignore-list.json

```diff
@@ -0,0 +1,35 @@
+{
+  "ec2" :
+    {
+      "input" : [],
+      "output" : []
+    },
+  "json" : {
+    "input" : [],
+    "output" : []
+  },
+  "json_1_0" : {
+    "input" : [],
+    "output" : []
+  },
+  "query" : {
+    "input" : [],
+    "output" : []
+  },
+  "rest-json" : {
+    "input" : [],
+    "output" : [
+      {"RestJsonDeserializesDenseSetMapAndSkipsNull": "Seeking clarification from Smithy"}
```

Comment:
This is a new test case that was added when I sourced updated protocol-tests - I THINK this was supposed to be omitted since I previously had convo with Kevin about sparse/dense maps for C2J. Clarifying with Kevin before addressing any changes if needed